.travis.yml

```diff
@@ -1,9 +1,11 @@
 # Travis continuous integration configuration file
 matrix:
   include:
+  - dist: xenial
```

Comment:
So, the fact that this starts with a dash means that it's an included run in the matrix all by itself. What you want to do is put `dist: xenial` right at the top of the file, methinks.